storage/src/main/java/tech/pegasys/teku/storage/server/kvstore/dataaccess/V4HotKvStoreDao.java

```diff
@@ -145,6 +150,58 @@ public HotUpdater hotUpdater() {
     return new V4HotUpdater(db, schema);
   }
 
+  @Override
+  public void ingest(
```

Comment:
I think all the same comments apply here as in the finalized dao - this can all just work at the `KvStoreAccessor` level.  In fact you may find that makes it pretty easy to share between the two implementations - just extract the schemas and accessors you need and then everything is likely to be the same for both hot and finalized.

Reply:
raised #4118 to cleanup the duplication.